scripts/us_census/acs5yr/subject_tables/common/resolve_geo_id.py

```diff
@@ -0,0 +1,65 @@
+"""
+Resolving place dcid given the GEOID string from data,census.gov.
+
+GEOID strings from census are typically 14 characters long that uniquely
+identifies a place based on summary level, geographic component and FIPS code.
+
+For example: The GEOID for California is 0400000US06, where
+040 - indicates summary-level (in this case, state-level)
+00  - indicates geographic variant
+00  - indicates geographic component
+US  - represents the United States
+06  - FIPS code for California State
+
+The code maps the place dcid based on the summary level (eg. state, zip code).
+The expected length of the fips_code for each summary level is tabulated in [1].
+The expected length of the fips_code can be used to debug issues in geoId resolution.
+
+Reference:
+1. https://www.census.gov/programs-surveys/geography/guidance/geo-identifiers.html
+2. https://mcdc.missouri.edu/geography/sumlevs/
+"""
+
+# Map for summary levels with expected geo prefix and FIPS code length
+_SUMMARY_LEVEL_PREFIX_MAP = {
```

Comment:
_SUMMARY_LEVEL_PREFIX_MAP
can add the 'US' and 'Geo' in the name if you have to make this global to use in tests.